libs/service-portal/settings/access-control/src/screens/Access/Access.treat.ts

```diff
@@ -28,3 +28,9 @@ export const closeButton = style({
   right: theme.spacing['1'],
   zIndex: 2,
 })
+
+export const tempTableDatePickerFix = style({})
+
+globalStyle(`${tempTableDatePickerFix} > div > table`, {
```

Comment:
Fyi: (I think) it's also possible to use "selectors:" in `style()` to select related elements, rather than using globalStyles.

Reply:
Yeah, apparently it's not supported (or at least not the last time I checked).
Can't use the ampersand on the left side in a CSS child combination with treat.
I asked them about it as a side question to an issue I submitted to treat: https://github.com/seek-oss/treat/issues/137